src/com/sun/jna/ptr/DoubleByReference.java

```diff
@@ -41,4 +43,8 @@ public double getValue() {
         return getPointer().getDouble(0);
     }
 
+    @Override
+    public String toString() {
+        return String.format("double@0x%x=%s", Pointer.nativeValue(getPointer()), Double.toString(getValue()));
```

Comment:
The explicit call to `Double#toString` is not necessary, String#format can directly be invoked with a primitive boolean